gateway/api/authentication.py

```diff
@@ -138,5 +138,23 @@ def authenticate(self, request):
                         )
                         provider.admin_groups.add(group)
                         logger.info("New provider created")
+                        return user, CustomToken(token.encode())
+
+                if token == settings.SETTINGS_AUTH_MOCK_TOKEN + "1":
```

Comment:
Why are we adding this, @akihikokuroda ?

Reply:
Oh, it's my mistake.  I take out changes in this file.